backend_new/src/app.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
+export class AppController {
```

Comment:
For this PR do we need to have all of the src files? These look like the default controller, service, etc that are generated for a nestjs app. If these aren't required could we limit this PR to the bare minimum?

Reply:
I feel like getting this set up with the default nest set up will make some of the other pieces easier later down the line, I'd rather leave these in place for now

if it feels like there is too much review in this though I can defo pull it out. your call!

Reply:
I feel like having the starter nest app makes a few steps later easier, so what I ended up doing is merging that into the feature branch, which shrinks this pr down to only parts related to prisma

lemme know if that works for you!